src/cpu/aarch64/jit_brgemm_conv_comp_pad_kernel.cpp

```diff
@@ -167,14 +168,14 @@ void jit_uni_brgemm_conv_comp_pad_kernel_t<isa>::khw_loop(const int icb,
         const int mb_tail, const int n_block) {
     Xbyak_aarch64::Label label_kw_loop, label_kw_end, label_kh_loop,
             label_kh_end;
-    add_imm(reg_kh_l, param1, GET_OFF(kh_l), X_TMP_0);
+    ldr(reg_kh_l, ptr(param1, static_cast<uint32_t>(GET_OFF(kh_l))));
```

Comment:
`add_imm` and `ldr` are very different, did this work before?

Reply:
Answered in: https://github.com/uxlfoundation/oneDNN/pull/3552#discussion_r2194547093